src/RestSharp/RestSharp.csproj

```diff
@@ -39,4 +39,7 @@
     <ItemGroup>
         <ProjectReference Include="$(RepoRoot)\gen\SourceGenerator\SourceGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
     </ItemGroup>
+    <ItemGroup>
```

Comment:
It doesn't sound right, adding this package.

Reply:
I think, we can remove it, seems like I added it by accident, is still building after removing it.

Reply:
If you commit the change, I'll mark this as resolved. Then I will merge your PR and figure out what to do with mutability.